index.html

```diff
@@ -4,16 +4,90 @@
 <head>
   <meta charset="utf-8">
 
-  <title>My Resume</title>
+  <title>Kelsy Resume</title>
 
+  
+  <link href="https://fonts.googleapis.com/css?family=Inconsolata" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
+  
+
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+
+  <section class= "top-section">
+    
+    <h1>Kelsy Wells</h1>
+
+    <nav>
+      <a href="#">Home</a>
+      <a href="#">About</a>
+      <a href="#">Portfolio</a>
+      <a href="#">Contact</a>
+    </nav>
+
+    <img class="face" src="kelsy2.jpg">
+
+    <h2>Cafe Manager </br>and </br>Part-Time Lambda School Student</h2>
+
+  </section>
+
+    <section id="why-hire-me">
+      <h3>Why should you hire me?</h3>
+      <p>Because I care about the work I do and I get the job done. 
+        I'm a fast learner and love experiencing new things. 
+        If you hire me I will be a dedicated and loyal employee who 
+        will do what it takes to help and grow with the company.</p>
+    </section>
+
+  <section class="mid-section">
+
+    <section id="experience">
+      <h3>Experience</h3>
+
+      <p>Dilettante Mocha Cafe </br>
```

Comment:
This works. Another way to break lines is to wrap each piece in "p" tags. Additionally, you can set the line height of the "p" tags using the "line-height" property.